conda/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set major = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_MAJOR )([^\)]+)') %}
+{% set minor = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_MINOR )([^\)]+)') %}
+{% set patch = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_PATCH )([^\)]+)') %}
+
+package:
+  name: gadgetron
+  version: {{ major.group(1) }}.{{ minor.group(1) }}.{{ patch.group(1) }}
+
+source:
+  path: ../
+
+requirements:
+  build:
+    - armadillo>=9.900.5
+    - boost=1.76.0
+    - cmake>=3.21.3
+    - cpr=1.7.2
+    - cuda-libraries-dev=11.6.1      # [linux64]
+    - cuda-libraries=11.6.1          # [linux64]
+    - cuda-runtime=11.6.1            # [linux64]
+    - cuda-nvcc=11.6.112             # [linux64]
+    - dcmtk=3.6.1
+    - fftw=3.3.9
+    - gcc_linux-64=9.4.0             # [linux64]
+    - gadgetron-python>=1.3.6
+    - gmock>=1.10.0
+    - gxx_linux-64=9.4.0             # [linux64]
+    - gtest=1.10.0
+    - howardhinnant_date=3.0.1
+    - ismrmrd=1.7.1
+    - ismrmrd-python>=1.9.8
+    - libblas=*=*mkl
+    - libcurl=7.79.1
+    - libxml2=2.9.12
+    - libxslt=1.1.33
```

Comment:
Not sure this is actually a dependency. I think it is a dependency for the siemens converter, but not the Gadgetron.